Jenkinsfile

```diff
@@ -21,48 +21,56 @@ node {
             Integer count = 0
             tests.eachWithIndex { test, index ->
                 echo "Scheduling ${test.Name}"
-                    testJobs[test.Id] = {
-                        node('usemango') {
-                            wrap([$class: "MaskPasswordsBuildWrapper", varPasswordPairs: [[password: '%useMangoApiKey%']]]) {
-                                dir ("${env.WORKSPACE}\\${tests[index].Id}") {
-                                    deleteDir()
+                testJobs[test.Id] = {
+                    node('usemango') {
+                        wrap([$class: "MaskPasswordsBuildWrapper", varPasswordPairs: [[password: '%useMangoApiKey%']]]) {
+                            dir ("${env.WORKSPACE}\\${tests[index].Id}") {
+                                deleteDir()
+                            }
+                            dir("${env.WORKSPACE}\\${tests[index].Id}") {
+                                ArrayList<LinkedHashMap<String, String>> scenarioList = tests[index].Scenarios
```

Comment:
Can we simplify this? This seems overly complicated. And isn't what you're getting back more like `ArrayList<KeyValuePair<String, String>>`?

Reply:
I am using intelliJ it tells you what object holds.

Plus this is the execution for confirmation
![image](https://github.com/infuse-consulting/jenkins/assets/60608318/c4d19ae4-1c7e-44c2-bf26-03f5107ba971)